# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apache/http-user-group.patch
# Copyright (C) 2006 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur httpd-2.2.8-orig/docs/conf/httpd.conf.in httpd-2.2.8/docs/conf/httpd.conf.in
--- httpd-2.2.8-orig/docs/conf/httpd.conf.in	2008-01-18 10:36:05.000000000 +0000
+++ httpd-2.2.8/docs/conf/httpd.conf.in	2008-01-18 10:37:26.000000000 +0000
@@ -63,8 +63,8 @@
 # It is usually good practice to create a dedicated user and group for
 # running httpd, as with most system services.
 #
-User daemon
-Group daemon
+User http
+Group http
 
 </IfModule>
 </IfModule>
